src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IVsSettingsStoreTester.cs

```diff
@@ -37,8 +36,7 @@ public int GetBool(string collectionPath, string propertyName, out int value)
         public int GetBoolOrDefault(string collectionPath, string propertyName, int defaultValue, out int value)
         {
             GetWithDefault(collectionPath, propertyName, 0, out value);
-            bool val;
-            value = VSConstants.S_OK != GetWithDefault(collectionPath, propertyName, false, out val) ? defaultValue : (val ? 1 : 0);
+            value = VSConstants.S_OK != GetWithDefault(collectionPath, propertyName, false, out bool val) ? defaultValue : (val ? 1 : 0);
```

Comment:
I think this would be better rewritten anyway regardless of the inline variable declaration - it's hard to read as it is, two ternary expressions in one line?